Public/Src/Deployment/Tests.Osx/test_runner.sh

```diff
@@ -14,6 +14,9 @@ chmod +x "$bxlSh"
 find "$MY_DIR/TestProj/tests" -name "*.TestProcess" -exec chmod +x {} \;
 find "$MY_DIR/TestProj/tests" -name "*CoreDump*" -exec chmod +x {} \;
 
+# Allow for up to 4mb of thread stack size
+export COMPlus_DefaultStackSize=400000
```

Comment:
move this to [`Public/Src/Sandbox/MacOS/scripts/bxl.sh`](https://github.com/microsoft/BuildXL/blob/master/Public/Src/Sandbox/MacOs/scripts/bxl.sh)

Reply:
Should this also be added when we launch the server process (in case a customer misses it?)

---
In reply to: [294366144](https://github.com/microsoft/BuildXL/pull/477#discussion_r294366144) [](ancestors = 294366144)